### PR TITLE
Add forms

### PR DESCRIPTION
**Pull request by anderCM**
## Project requirements
- Create a method called current_user in your ApplicationController to make current user data available to all controllers.
  - It will return the first user from the database.
  - Don't worry about proper authentication at this time. You will add it in the upcoming projects.
- Create forms to perform the following functions:
  - Creates a Post on behalf of the current_user (use the method that you created in your ApplicationController).
  - Create a comment on behalf of the current_user (use the method that you created in your ApplicationController).
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -1,2 +1,19 @@
 class CommentsController < ApplicationController
+  def create
```
Comment (codecaiine) [inline app/controllers/comments_controller.rb:6]: - [ ] Good job so far 👏  but According to the project requirement user needs to be able to create a comment on behalf of the current_user, for that reason it should have a method **new**  to get a form, and a form  **views/comments/new.html.erb** file with input for creating comments and also make sure that we have all the routes. Please fix it in order to make your app works as expected 👍

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @anderCM , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Good form setup ✔️ 
- No rubocop errors ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/comments_controller.rb (hunk 1 above)
Comment (anderCM) [reply app/controllers/comments_controller.rb:6]: Hi @codecaiine I get your point according to get a new view in `app/views/comments/new.html.erb` and the method `new` in `CommentsController` it's not listed in the requirements, it just says:  `Create a comment on behalf of the current_user`. So it doesn't mention that it's strictly necessary, but it's necessary to create a comment on behalf of the current_user.
It's been fulfilled in the method `create` where I am assigning `current_user` to `@comment.user`:
```
def create
    @post = Post.find(params[:post_id])
    @comment = @post.comments.new(comment_params)
    @comment.user = current_user
    if @comment.save
      @comment.update_counter
      redirect_to user_post_path(current_user.id, @post.id), notice: 'You created a comment!'
    else
      render :new, status: :unprocessable_entity
    end
  end
  ```

Comment (codecaiine) [reply app/controllers/comments_controller.rb:6]: Hi, @anderCM  Thank you to reach out to me. In the project requirement, you are supposed to create a form for both POST AND COMMENT and In order to get this form you should have a method **def new** and **app/views/comments/new.html.erb** file inside the comment controller like the one you did in your Post controller.

Comment (anderCM) [reply app/controllers/comments_controller.rb:6]: @codecaiine  Thank you for clarifying your point. I will work on that

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi Team, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark:  Clean and Organized code
:heavy_check_mark: Good communication with code reviewer

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

#### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**